variables.tf

```diff
@@ -363,6 +363,28 @@ variable "security_group_rules" {
   }
 }
 
+variable "clean_default_security_group" {
+  description = "Remove all rules from the default VPC security group (less permissive)"
+  type        = bool
+  default     = false
+}
+
+variable "clean_default_acl" {
+  description = "Remove all rules from the default VPC ACL (less permissive)"
+  type        = bool
+  default     = false
+}
+
+variable "ibmcloud_api_visibility" {
+  description = "IBM Cloud API visibility used for internal scripts. Must be 'public', 'private', or 'public-and-private'"
```

Comment:
The word "internal" might be confusing here (since this is a public module). Perhaps say "used for scripts run by this module" ?